var/spack/repos/builtin.mock/packages/test-missing-test/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+from spack.package import run_after
+
+
+class TestMissingTest(Package):
```

Comment:
I don't understand what the missinig test method has to do with the test using this package... was this repurposed from some other work?

Reply:
It's been a while but I am assuming it was for test coverage.

Reply:
Maybe rename to `TestInstallCallbackTest`